django_project/users/templates/users/activate_account.html

```diff
@@ -0,0 +1,7 @@
+{% autoescape off %}
+Hi {{ user.first_name }},
+Please click on the link to confirm your registration if the following information is correct.
+BC Email: {{ user.username }}
+Eagle ID: {{ user.profile.eagle_id }}
+http://{{ domain }}{% url 'activate' uidb64=uid token=token %}
+{% endautoescape %}
```

Comment:
I just realized, maybe we can have them reply to these emails if their information is not correct? For example, if they entered their eagle id incorrectly. So then we could add a final part to this message that says something like,

"If this information is not correct, please respond to this email and the site administrator will assist you."

Reply:
They can modify it themselves in the profile page.